README.md

```diff
@@ -24,8 +24,11 @@ $ python setup.py install # install wallet-sdk-py
 
 ### Run unit test
 
-The following command will run unit test.
+The following command will run unit test
 
 ```sh
 $ pytest
 ```
+
+### Wallet APIs
+Details of Wallet APIs please refer to [Wallet APIs Documentation](http://www.arxanfintech.com/infocenter/html/development/wallet.html#wallet-platform-ref)
```

Comment:
you should provide SDK manual and procedure of create wallet account, create asset and etc, instead just provide APIs doc simply here.